src/mpcCoreKit.ts

```diff
@@ -1465,7 +1513,8 @@ export class Web3AuthMPCCoreKit implements ICoreKit, IMPCContext {
    */
   private async copyOrCreateShare(newFactorTSSIndex: number, newFactorPub: Point) {
     this.checkReady();
-    const tssData = this.getTssData();
+    const firstKeyType = this.getSupportedCurveKeyTypes()[0];
```

Comment:
Since you use this a lot, I think you should create a dedicated function for it.
What's the logic behind always using the first key type here?

Reply:
I are trying to get tssIndex from getTssShare or factorPub from getTssData.
both curves should have the same tssIndex (for same factorKey) and factorPub
Using either curve's keyType will get the same value.

using the first as instance might configured with single curve keytype only